### PR TITLE
Add Dojo Library VS snippets link (Lombiq Technologies: OCORE-33)

### PR DESCRIPTION
**Pull request by Piedone**
I'll also demo the snippets on Tuesday.

### CODE REVIEW

**Comment by ns8482e** [comment]
How about adding these to separate page Additional Resources, where community contributors can be listed ?

**Comment by Piedone** [comment]
There is the Resources page for it. I've added these to here because this section is specifically about using Orchard with VS.

**Comment by agriffard** [comment]
I also suggest to add this link somewhere in the Resources section.

**Comment by Piedone** [comment]
What about having a Development Tools page under Resources that provides suggestions on what to use for working with OC? Since I don't think any of the pages there are applicable. I'd list VS, VS Code, and Rider for C# developers. I have no experience with anything for just Liquid/CSS/JS development, should that be a target demographics too.

**Comment by agriffard** [comment]
> What about having a Development Tools page under Resources

Good idea.
We could list 'SQL browser lite' to inspect yessql.db files and smtp4dev if you want to test to send an email.

**Comment by Piedone** [comment]
I've added the Development Tools page, what do you think? Also, since I only work on Windows I don't have any experience with IDEs on Mac (or Linux distros), could you please add anything that you deem important @deanmarcussen?

File: src/docs/resources/development-tools/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Development Tools
+
+What tools do we recommend to build your app with Orchard Core, or work on Orchard itself? In  the end, this is up to your personal preference since as long as you can edit source files and build the app you can use any tool on any platform that .NET Core supports. Below are some tips to get you going for the general editing experience as well as for other useful tools.
```
Comment (agriffard) [inline src/docs/resources/development-tools/README.md:3]: In  the end => In the end. 
Remove the extra space.

Comment (Piedone) [reply src/docs/resources/development-tools/README.md:3]: Thanks, removed it.

**Pull request merged**